src/it/scala/loamstream/loam/intake/CsvTransformationTest.scala

```diff
@@ -321,26 +323,17 @@ object CsvTransformationTest {
           tag("makeCSV").
           in(inputDataFile)
       
-      produceSchemaFile(schemaFile).
-          from(columns: _*).
-          tag("makeSchemaFile")
-      
-      produceListFiles(dataListFile, schemaListFile).
-          from(mungedDataFile, schemaFile).
-          tag("makeListFiles").
-          in(mungedDataFile, schemaFile)
-      
       val aggregatorConfigFile = store(aggregatorConfigFilePath)
       
       val sourceColumns = aggregator.SourceColumns(
           marker = Loam.ColumnNames.VARID,
           pValue = Loam.ColumnNames.PValue,
-          zScore = Loam.ColumnNames.OddsRatio,
-          stderr = Loam.ColumnNames.SE,
-          beta = Loam.ColumnNames.OddsRatio,
-          oddsRatio = Loam.ColumnNames.OddsRatio,
-          eaf = Loam.ColumnNames.EAF,
-          maf = Loam.ColumnNames.MAF)
+          zScore = Some(Loam.ColumnNames.OddsRatio),
```

Comment:
Not sure if this is right/intentional (zScore mapping to OddsRatio)? Likewise below is beta -> oddsRatio.

Reply:
These mappings are all totally arbitrary, and date from when I knew even less about the underlying data than I do now.  Perhaps the columns should just be called foo, bar, baz, etc to emphasize this.  For now, this test exists to validate that a known transformation can be applied to known data, producing expected output.